### ISSUE TITLE
Get-WhatsNew should return a single string instead of an array of strings

### ISSUE THREAD

**Issue by daxian-dbw** [PowerShell/whatsnew#23]
### Prerequisites

- [X] Write a descriptive title.
- [X] Make sure you are able to repro it on the latest version
- [X] Search the existing issues.

### Steps to reproduce

`Get-WhatsNew` should return a single string instead of an array of strings.

There seems now point to return an array of strings from it. Also, returning an array of strings prevents `Get-WhatsNew | Show-Markdown` from working.

### Expected behavior

```console
fff
```

### Actual behavior

```console
fff
```

### Environment data

```PowerShell
PS:17> Get-Module Microsoft.PowerShell.WhatsNew

ModuleType Version    PreRelease Name                                ExportedCommands
---------- -------    ---------- ----                                ----------------
Script     0.4.0                 Microsoft.PowerShell.WhatsNew       Get-WhatsNew
```
```


### Version

0.4.0

### Visuals

_No response_

**Comment by pwahlmueller** [comment]
The first thing I wanted to do, is to find out, what are the new things in the last month. 

But it was a string and no Object, so I could not.

So I would love to see some addintioal information like the realease date. Also a reference to the issue would be nice, I think.

**Comment by mklement0** [comment]
@daxian-dbw, line-by-line output can be useful with `Select-String` operations, e.g.:

```powershell
Get-WhatsNew | Select-String 'PSFeedback' -Context 0, 10
```

Wouldn't it be better to amend `Show-Markdown` to simply collect all input strings first, analogous to what `ConvertFrom-Json` does, for instance, which is the implied suggestion in #20951?

This would also support scenarios such as `Get-Content ... | Show-Markdown` (even though use of `-Path` / `-LiteralPath` is better).

**Comment by mklement0** [comment]
@pwahlmueller, providing metadata for filtering is also possible with `[string]` output, using instance ETS properties (which is what `Get-Content` does, for instance).

As for your desire to filter by date _range_ - that doesn't really apply, given that `Get-WhatsNew` is focused on _releases_.

However, knowing when a given version was released would be helpful, not least as part of the _text_.

If you'd like to see metadata attached to the output strings, I suggest opening a  feature request in which you can flesh out the specifics.

**Comment by ArmaanMcleod** [comment]
Maybe we could introduce a `-AsString` parameter so the user can opt in to return output as single string.

**Comment by mklement0** [comment]
@ArmaanMcleod, but if `Show-Markdown` collected all its input first, there'd be no need for such a switch.

Do you anticipate any problems with the suggested `Show-Markdown` change?

**Comment by ArmaanMcleod** [comment]
@mklement0 Yeah I think it's preferable to fix this in `Show-Markdown` to collect strings from `-InputObject`.

I was just suggesting the `-AsString` parameter here if we couldn't change the output type for `Get-WhatsNew` and the fix needed to be with this cmdlet instead of fixing `Show-Command`. The extra parameter would be similar to `Get-Content  -Raw` 😂.

Many options I think though. Probably needs input and final say from PowerShell team on what solution we should do.
